composer.json

```diff
@@ -14,8 +14,8 @@
         "composer-plugin-api": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit":   "^5.0",
-        "composer/composer": "^1.0.0-ALPHA8@ALPHA"
+        "phpunit/phpunit":           "^5.2",
```

Comment:
Should be changed to `^5.2.8` once https://github.com/sebastianbergmann/phpunit/pull/2082 is merged